test/libsolidity/syntaxTests/fallback/default_visibility.sol

```diff
@@ -3,4 +3,5 @@ contract C {
     function() {}
 }
 // ----
+// SyntaxError: (90-103): No visibility specified. Did you intend to add "public"?
 // TypeError: (90-103): Fallback function must be defined as "external".
```

Comment:
@chriseth This turned out to be a bit confusing. What about suggesting ``external`` for fallback and interface functions?